documentation/mkdocs.yml

```diff
@@ -7,28 +7,29 @@ nav:
         - 'Overview': infrastructure/overview.md
         - 'Custom Jenkins Setup': infrastructure/customjenkins.md
     - 'Pipelines':
+        - 'ABAP Environment pipeline':
+          - 'Introduction': pipelines/abapEnvironment/introduction.md
+          - 'Configuration': pipelines/abapEnvironment/configuration.md
         - 'General purpose pipeline':
-            - 'Introduction': stages/introduction.md
-            - 'Examples': stages/examples.md
-            - 'Stages':
-                - 'Init Stage': stages/init.md
-                - 'Pull-Request Voting Stage': stages/prvoting.md
-                - 'Build Stage': stages/build.md
-                - 'Additional Unit Test Stage': stages/additionalunittests.md
-                - 'Integration Stage': stages/integration.md
-                - 'Acceptance Stage': stages/acceptance.md
-                - 'Security Stage': stages/security.md
-                - 'Performance Stage': stages/performance.md
-                - 'Compliance': stages/compliance.md
-                - 'Confirm Stage': stages/confirm.md
-                - 'Promote Stage': stages/promote.md
-                - 'Release Stage': stages/release.md
+          - 'Introduction': stages/introduction.md
```

Comment:
If you change the indent (👍) please do for all levels.

Reply:
Adapted formatting to make it similar to the other sections of the file (two tabs)